### PR TITLE
fix: faithfulness for unrelated answering

### PR DESCRIPTION
**Pull request by shahules786**

### CODE REVIEW

**Comment by shahules786** [comment]
```
data={'question': 'What is the economic significance of New York City?',
 'ground_truths': ["New York City's economic significance is vast, as it serves as the global financial capital, housing Wall Street and major financial institutions. Its diverse economy spans technology, media, healthcare, education, and more, making it resilient to economic fluctuations. NYC is a hub for international business, attracting global companies, and boasts a large, skilled labor force. Its real estate market, tourism, cultural industries, and educational institutions further fuel its economic prowess. The city's transportation network and global influence amplify its impact on the world stage, solidifying its status as a vital economic player and cultural epicenter."],
 'answer': 'idk what the fuck.',
 "contexts":["New York City's most important economic sector lies in its role as the headquarters for the U.S. financial industry, metonymously known as Wall Street. The city's securities industry continues to form the largest segment of the city's financial sector and is an important economic engine. Many large financial companies are headquartered in New York City, and the city is also home to a burgeoning number of financial startup companies."]
}
  
faithfulness.score_single(data)
```

**Comment by shahules786** [comment]
@Yongtae723 Would you like to play with it and see if it holds up to your adversarial testing? Feel free to modify if not :)

File: src/ragas/metrics/_faithfulness.py [hunk 1]
```diff
@@ -96,6 +96,17 @@
     }}
 ]
 
+Context:
+Albert Einstein was a German-born theoretical physicist who is widely held to be one of the greatest and most influential scientists of all time.
+statement_1: Nil
+Answer:
+[
+     {{
+        "statement_4": "Nil",
```
Comment (Yongtae723) [inline src/ragas/metrics/_faithfulness.py:105]:
```suggestion
        "statement_1": "Nil",
```

I guess

**Review by Yongtae723 — COMMENTED** [review]
I haven't check but I feel LGTM except minor point!

I will report after testing

**Comment by shahules786** [comment]
Thank you @Yongtae723

**Comment by Yongtae723** [comment]
I tested and confirmed it works well

**Pull request merged**